[IMP] mail: from filter ICP missing part (#13134)
odoo/documentation: content/applications/general/email_communication/email_servers_outbound.rst

   If the database subdomain is `company-name.odoo.com` and all mailing configurations are the
   default ones, all emails will be sent from `notifications@company-name.odoo.com`.

> .. _email-outbound-default-from-filtering:
> 
> This configuration is done by the system parameter `mail.default.from_filter`.
> In case the sender's domain do not match the value of this parameter, the notification address
> will be used.
> Multiple values can be defined, comma-separated, domains or full email addresses.
> Once an outgoing mail server is active, the parameter is is no longer considered the value used
> will be the :ref:`email-outbound-different-servers-personalized-from-filtering` of the mail server.
> 

Reviewer: Suggesting to use a note admonition here, along with some slight rewording for clarity. 

We can also use the existing `email-outbound-default` anchor for this section.
```suggestion
.. note::
   This configuration is handled by the `mail.default.from_filter` system parameter. In cases where
   the sender's domain does not match the value of this parameter, the notification address is used
   instead.

   Multiple values can be defined in this system parameter: comma-separated, domains, or full email
   addresses are all allowed.

   Once an :ref:`outgoing mail server is configured
   <email-outbound-different-servers-personalized>`, the system parameter is no longer considered
   and the value used is the :ref:`FROM filtering
   <email-outbound-different-servers-personalized-from-filtering>` of the mail server.
```

   higher the priority is). Failing to do so, the first server is determined by the servers' names,
   using alphabetical order.

> - If there is no mail server, Odoo will rely on the :ref:`system parameter <email-outbound-default-from-filtering>` value.
> 

Reviewer: Slight correction to avoid future tense, update the anchor that is used, and break the line before 100 chars :)
```suggestion
- If there is no mail server, Odoo relies on the :ref:`system parameter <email-outbound-default>`
  value.
  
```
